package.json

```diff
@@ -25,6 +25,9 @@
       "*.ts"
     ]
   },
+  "eslintConfig": {
+    "extends": "standard"
```

Comment:
What does this do?

Reply:
makes my editor (vscode) format code when files are saved as opposed to discovering the issues after I push